plugin/build.gradle.kts

```diff
@@ -243,10 +231,87 @@ java {
   withJavadocJar()
 }
 
-tasks.shadowJar.configure {
-  archiveClassifier.set("")
+if(project.properties["rfg.skipJavadoc"].toString().toBoolean()) {
+  tasks.named("javadoc").configure { enabled = false }
```

Comment:
Instead of disabling it here, wrap the `withJavadocJar()` call above in the if

Reply:
That would also require wrapping all the lines referencing `javadocJar` later in the file, I decided to do it this way to keep it more self-contained.